UPGRADE.md

```diff
@@ -106,6 +106,10 @@ or any public API on instantiated objects.
 Please implement the `Doctrine\ORM\Repository\RepositoryFactory` interface instead of extending
 the `Doctrine\ORM\Repository\DefaultRepositoryFactory`.
 
+## BC BREAK: `Doctrine/\ORM\Persisters\BasicEntityPersister::getSelectConditionStatementColumnSQL` has been merked as `private`
+
+The `getSelectConditionStatementColumnSQL` has been removed (marked as `private`) from the API.
```

Comment:
Fair enough, though the persisters really changed radically (even their namespace) :|